src/main/java/org/checkerframework/checker/dependencyinjection/DependencyInjectionAnnotatedTypeFactory.java

```diff
@@ -105,6 +105,67 @@ protected boolean isToMethod(Tree methodTree) {
     return TreeUtils.isMethodInvocation(methodTree, this.toMethods, this.getProcessingEnv());
   }
 
+  private void handleBindMethodInvocation(Node methodArgumentNode) {
```

Comment:
Both this method and `handleToMethodInvocation` should have a Javadoc comment that explains their precondition---that the argument is a specific method invocation---and how that should be checked (call `isBindMethod()` or `isToMethod()`).